lib/stages/rawFileWrite.cpp

```diff
@@ -36,6 +39,8 @@ rawFileWrite::rawFileWrite(Config& config, const std::string& unique_name,
     base_dir = config.get<std::string>(unique_name, "base_dir");
     file_name = config.get<std::string>(unique_name, "file_name");
     file_ext = config.get<std::string>(unique_name, "file_ext");
+    _num_frames_per_file = config.get_default<uint32_t>(unique_name, "num_frames_per_file", 1);
+    prefix_hostname = config.get_default<bool>(unique_name, "prefix_hostname", true);
```

Comment:
Can we clean up these names to include the `_` prefix while we are at it?  I don't like seeing mixed standards.